### PR TITLE
Dont copy .nupkg as .symbol.nupkgs

### PR DESCRIPTION
**Pull request by JohnTortugo**
Closes: https://github.com/dotnet/arcade/issues/4519

Some builds produce .nupkgs that are only "resources" and don't contain any PDBs. Adding option to let the build forward that information to Arcade SDK.

I've to still test this in the runtime-assets repository.

### CODE REVIEW

File: src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj [hunk 1]
```diff
@@ -18,6 +18,8 @@
     DotNetSignType                  Specifies the signing type: 'real' (default), 'test'.
     DotNetRuntimeSourceFeed         Storage account to lookup for the .Net runtime files
     DotNetRuntimeSourceFeedKey      In case the storage account to fetch the .Net runtime files are private this should contain a SAS token.
+    DotNetNupkgContainsPortablePdb  Will the generated NuGet packages contain Portable PDBs.
```
Comment (tmat) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj:21]: >DotNetNupkgContainsPortablePdb [](start = 4, length = 30)

I think this property should be set in Publishing.props file that's imported by Publish.proj, not here.

**Comment by tmat** [comment]
>   <Import Project="$(RepositoryEngineeringDir)Publishing.props" Condition="Exists('$(RepositoryEngineeringDir)Publishing.props')" />

Publishing.props should set the property that controls this. 

I'd call the property `AutoGenerateSymbolPackages`. That's what we are doing here. If it's disabled then we don't generate any symbol packages, we just publish those that are already created by the repo.

---
Refers to: src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj:19 in a1004d3. [](commit_id = a1004d3820d4309565603b4884bafd4ac99f2453, deletion_comment = False)

File: src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj (hunk 1 above)
Comment (JohnTortugo) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj:21]: Definitely. Makes total sense.

**Comment by JohnTortugo** [comment]
> I'd call the property `AutoGenerateSymbolPackages`. That's what we are doing here. If it's disabled then we don't generate any symbol packages, we just publish those that are already created by the repo.

I named the parameter like I did because of a comment in Publish.proj: _"If a symbol package doesn't exist yet we assume that the regular package contains Portable PDBs."_. Anyway, I don't have any problem renaming it.

**Comment by JohnTortugo** [comment]
Validated here: https://github.com/dotnet/runtime-assets/pull/68

**Pull request merged**